src/clients/api/mutations/useApproveToken/index.ts

```diff
@@ -4,17 +4,18 @@ import { TokenId } from 'types';
 import { queryClient, approveToken, IApproveTokenInput, ApproveTokenOutput } from 'clients/api';
 import FunctionKey from 'constants/functionKey';
 import { useTokenContract } from 'clients/contracts/hooks';
+import setCachedTokenAllowanceToMax from './setCachedTokenAllowanceToMax';
 
 const useApproveToken = (
-  { assetId }: { assetId: TokenId },
+  { tokenId }: { tokenId: TokenId },
```

Comment:
nit: Noticing this name change, I think we should choose between asset and token for referencing assets. I have been going with asset recently because that is what they are called on the frontend.

Reply:
Having two names  makes sense if they have two different definitions. I don't think extending the returned token data is enough to warrant a new name or definition. 

We should define a layer where we validate and format data we receive from the backend. This layer would be where we would modify any naming conventions for example switching from camel to snake case for example. 

To me token and asset are interchangeable and refer to the same thing so I would pick one but if we can define what they each are then we can use both.